### PR TITLE
Fix iOS Custom AspectRatio

### PR DESCRIPTION
**Pull request by chekodev**

### CODE REVIEW

File: ImageCropper/ImageCropper.iOS/ImageCropperImplementation.cs [hunk 1]
```diff
@@ -32,6 +32,8 @@ public void ShowFromFile(ImageCropper imageCropper, string imageFile)
             if(imageCropper.AspectRatioX > 0 && imageCropper.AspectRatioY > 0)
             {
                 cropViewController.AspectRatioPreset = TOCropViewControllerAspectRatioPreset.Custom;
+                cropViewController.ResetAspectRatioEnabled = false;
+                cropViewController.AspectRatioLockEnabled = true;
                 cropViewController.CustomAspectRatio = new CGSize(imageCropper.AspectRatioX, imageCropper.AspectRatioY);
```
Comment (stormlion227) [inline ImageCropper/ImageCropper.iOS/ImageCropperImplementation.cs:37]: Which issues is there?

Comment (chekodev) [reply ImageCropper/ImageCropper.iOS/ImageCropperImplementation.cs:37]: When set the Custom AspectRatio (for example: X = 1, Y = 1), in Android all is well, the aspectRatio is maintained while the user cropping the picture. But in IOS the user can change the aspectRatio, simply dragging the clipping area or in the configuration bar

Comment (stormlion227) [reply ImageCropper/ImageCropper.iOS/ImageCropperImplementation.cs:37]: Thanks for your contribution. :)

Comment (chekodev) [reply ImageCropper/ImageCropper.iOS/ImageCropperImplementation.cs:37]: You are welcome

Comment (kumarkumar217) [reply ImageCropper/ImageCropper.iOS/ImageCropperImplementation.cs:37]: when i  have added the image cropper as project the aspect ratio working fine, but when i am trying to use with nuget package. then aspect ratio is not working.

**Pull request merged**